packet/routes/shared.py

```diff
@@ -1,56 +1,60 @@
+"""
+Routes available to both freshmen and CSH users
+"""
+
 from flask import render_template, redirect
 
 from packet import auth, app
-from packet.ldap import ldap_is_eboard
-from packet.member import current_packets
-from packet.packet import get_number_signed, signed_packet, get_freshman, \
-    get_number_required_off_floor, get_number_required_on_floor
-from packet.packet import get_signatures, get_upperclassmen_percent
-from packet.utils import before_request
+from packet.utils import before_request, packet_auth
+from packet.models import Packet
 
 
-@app.route('/logout')
+@app.route("/logout/")
 @auth.oidc_logout
 def logout():
-    return redirect("/")
+    return redirect("http://csh.rit.edu")
```

Comment:
Is there a reason this now redirects to the pubsite? This is different behavior from all of our other apps.

Reply:
I was trying to get keycloak to not display the "invalid redirect uri" error on logout but I guess it didn't work and I forgot to revert. I can change it back if you'd prefer but it doesn't really change anything right now.